### PR TITLE
Replace main page with responsive one

### PR DESCRIPTION
**Pull request by anxolerd**
- Use unused index.html template for that
- Materializecss framework for layout
- Everything from CDNs

image

### CODE REVIEW

File: src/GDGUkraine/controller.py [hunk 1]
```diff
@@ -37,7 +37,8 @@ class Root:
     @cherrypy.expose
     # @render(template = 'gdg.org.ua_old.html', page_id = 'about')
     def index(self, **kwargs):
-        tmpl = get_template('gdg.org.ua_old.html')
+        # tmpl = get_template('gdg.org.ua_old.html')
```
Comment (webknjaz) [inline src/GDGUkraine/controller.py]: -> `/map`

File: src/GDGUkraine/templates/index.html [hunk 2]
```diff
@@ -1,17 +1,103 @@
...
-            $.ajaxSetup({ scriptCharset:"utf-8",
-                            contentType:"application/json; charset=utf-8" });
-            $.post(url, $.toJSON(data), callback, "json");
-        }
-    </script>
-{%- endblock head %}
-{% block content -%}
-    Hello World!
-{%- endblock content %}
+<!DOCTYPE html>
```
Comment (webknjaz) [inline src/GDGUkraine/templates/index.html]: why not inherit base?

File: src/GDGUkraine/controller.py (hunk 1 above)
Comment (anxolerd) [inline src/GDGUkraine/controller.py]: No

File: src/GDGUkraine/templates/index.html (hunk 2 above)
Comment (anxolerd) [inline src/GDGUkraine/templates/index.html]: This was easier.
I prefer to have unrelated pages separated.

File: static/js/map.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      center: new google.maps.LatLng(clat, clng),
+      zoom: zoom,
+      mapTypeId: google.maps.MapTypeId.ROADMAP,
+      scrollwheel: false,
+    };
+    var map = new google.maps.Map(node, mapOptions);
+    places.forEach(function(p) {
+      var marker = new google.maps.Marker({
+        position: new google.maps.LatLng(p.lat, p.lng),
+        map: map,
```
Comment (anxolerd) [inline static/js/map.js]: we have no babel for that.

File: static/js/map.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+  var initializeMap = function(node, attrs) {
+    var clat = attrs.clat ? parseFloat(attrs.clat) : DEFAULT_PARAMS.clat;
+    var clng = attrs.clng ? parseFloat(attrs.clng) : DEFAULT_PARAMS.clng;
+    var places = attrs.places ? JSON.parse(attrs.places) : DEFAULT_PARAMS.places;
+    var zoom = attrs.zoom ? parseFloat(attrs.zoom) : DEFAULT_PARAMS.zoom;
+
+    var mapOptions = {
+      center: new google.maps.LatLng(clat, clng),
+      zoom: zoom,
```
Comment (anxolerd) [inline static/js/map.js]: we have no babel for that

**Comment by anxolerd** [comment]
- > Issue found: Expected property shorthand. (object-shorthand)

     Unless we have babel and build process for frontend, this won't be fixed.

- > Issue found: 'google' is not defined. (no-undef)

     Google Maps library has to be loaded separately (unfortunately). I can do nothing with it.
- > Issue found: Function 'anonymous' has a complexity of 5. (complexity)

     Function is pretty straightforward. The only "complex" place is parsing the inputs.

**Comment by anxolerd** [comment]
@webknjaz pls review it again.
I want to update the main page ASAP, cause the current landing is a shame.

File: src/GDGUkraine/controller.py [hunk 5]
```diff
@@ -35,12 +35,23 @@ def wrapper(obj):
...
     def index(self, **kwargs):
-        tmpl = get_template('gdg.org.ua_old.html')
+        tmpl = get_template('index.html')
         return tmpl.render(
-            places=api.get_all_gdg_places(cherrypy.request.orm_session,
-                                          filtered=True))
+            places=[
+                {
+                    'name': place.name,
+                    'lat': place.geo.split(',')[0],
```
Comment (webknjaz) [inline src/GDGUkraine/controller.py]: you do this twice. don't.

**Pull request merged**